### PR TITLE
chore: change the constant number to const variable

### PR DESCRIPTION
**Pull request by fwx5618177**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
- Change the number to constant variable.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Review by shtripat — APPROVED** [review]
lgtm

File: cmd/bucket-lifecycle.go [hunk 1]
```diff
@@ -144,10 +144,14 @@ func (es *expiryState) enqueueByNewerNoncurrent(bucket string, versions []Object
 
 var globalExpiryState *expiryState
 
+const (
+	ChannelCapacity = 100000
```
Comment (klauspost) [inline cmd/bucket-lifecycle.go]: Shouldn't be exported and should have a descriptive name.

File: cmd/bucket-replication.go [hunk 2]
```diff
@@ -1687,6 +1687,9 @@ const (
 
 	// LargeWorkerCount is default number of workers assigned to large uploads ( >= 128MiB)
 	LargeWorkerCount = 10
+
+	// Default number of Replication Pool
+	ReplicationDefault = 100000
```
Comment (klauspost) [inline cmd/bucket-replication.go]: Descriptive name.

**Review by klauspost — CHANGES_REQUESTED** [review]

File: cmd/bucket-lifecycle.go (hunk 1 above)
Comment (fwx5618177) [reply cmd/bucket-lifecycle.go]: Great! I change these name.

`ChannelCapacity ` to `MaximumExpiryTasks`, could describe more clearly and exactly what's this constant value. It means that maximum expiry tasks.
`ReplicationDefault ` to `DefaultReplicationPoolSize`, defines the default size of the replication pool.

**Comment by harshavardhana** [comment]
Please resolve the conflicts @fwx5618177 thank you

**Comment by fwx5618177** [comment]
> Please resolve the conflicts @fwx5618177 thank you

Done.

**Review by klauspost — APPROVED** [review]

**Comment by fwx5618177** [comment]
I don't know why always mentioned errors

**Comment by harshavardhana** [comment]
A random flyby commit doesn't add anything closing, as many such places need fixing. We would prefer this to be done by the MinIO team rather than externally.

**Comment by fwx5618177** [comment]
> A random flyby commit doesn't add anything closing, as many such places need fixing. We would prefer this to be done by the MinIO team rather than externally.

Okie, but I prefer to join our community to optimize minIO

**Pull request closed without merging**